server/polar/issue/endpoints.py

```diff
@@ -182,37 +182,23 @@ async def lookup(
 
         client = get_polar_client()
 
-        try:
```

Comment:
Here is a case where we do not need to manually catch the exception anymore.

I made the change in some places, but other places probably need to move to this new "paradigm".